.github/workflows/testing.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           VALIDATE_NATURAL_LANGUAGE: false
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_KUBERNETES_KUBEVAL: false
+          FILTER_REGEX_EXCLUDE: ".*/charts/kangal/templates/(serviceaccount|deployment).yaml"
```

Comment:
just exclude all the chart templates for now with the TODO - we'll find out hot to properly fix it later

Reply:
updated thanks